source/platforms/_tests/fixtures/bbc-dsl-input.json

```diff
@@ -0,0 +1,2549 @@
+      },
+      {
+        "sha": "097d11d21257881ba71157392dfa0f5bf094dc9f",
+        "author": {
+          "name": "Ryo",
+          "email": "ryo@ryo.com",
+          "date": "2019-05-13T11:44:42+00:00"
+        },
+        "committer": {
+          "name": "Ryo",
+          "email": "ryo@ryo.com",
+          "date": "2019-05-13T11:44:42+00:00"
+        },
+        "message": "Merged in fix/points-history-text-style (pull request #169)\n\nPointsHistoryCard adding a new textstyle\n\nApproved-by: DangerCI <danger@ci.com>\n",
+        "tree": null,
+        "parents": [
+          "04eadff2879786f21dd60253c1a5c49a1b72d08b",
+          "5b70bec17deefe19c82782f8bbfbb7e1a7533ac8"
+        ],
+        "url": "https://bitbucket.org/foo/bar/commits/097d11d21257881ba71157392dfa0f5bf094dc9f"
+      },
+      {
+        "sha": "5b70bec17deefe19c82782f8bbfbb7e1a7533ac8",
+        "author": {
+          "name": "Furyou81",
+          "email": "lfujimot@student.42.fr",
+          "date": "2019-05-13T11:41:13+00:00"
+        },
+        "committer": {
+          "name": "Furyou81",
+          "email": "lfujimot@student.42.fr",
+          "date": "2019-05-13T11:41:13+00:00"
+        },
+        "message": "merge\n",
+        "tree": null,
+        "parents": [
+          "0f20860ab9b1446d39d6aff4df90d90060ccc329",
+          "04eadff2879786f21dd60253c1a5c49a1b72d08b"
+        ],
+        "url": "https://bitbucket.org/foo/bar/commits/5b70bec17deefe19c82782f8bbfbb7e1a7533ac8"
+      },
+      {
+        "sha": "04eadff2879786f21dd60253c1a5c49a1b72d08b",
+        "author": {
+          "name": "Hello Core",
+          "email": "foo@bar.com",
+          "date": "2019-05-13T11:07:20+00:00"
+        },
+        "committer": {
+          "name": "Hello Core",
+          "email": "foo@bar.com",
+          "date": "2019-05-13T11:07:20+00:00"
+        },
+        "message": "Merged in feature/app-icon-temp (pull request #171)\n\nFeature/app icon temp\n\nApproved-by: DangerCI <danger@ci.com>\nApproved-by: Ryo <ryo@ryo.com>\n",
+        "tree": null,
+        "parents": [
+          "d802defdf0e6b4f85e01e48f2576b5f4ade8c77d",
+          "301b16bbd02e1f2107ce3e50812afd584c5a99e0"
+        ],
+        "url": "https://bitbucket.org/foo/bar/commits/04eadff2879786f21dd60253c1a5c49a1b72d08b"
+      },
+      {
+        "sha": "301b16bbd02e1f2107ce3e50812afd584c5a99e0",
+        "author": {
+          "name": "Danger",
+          "email": "danger@abc.com",
+          "date": "2019-05-13T11:04:13+00:00"
+        },
+        "committer": {
+          "name": "Danger",
+          "email": "danger@abc.com",
+          "date": "2019-05-13T11:04:13+00:00"
+        },
+        "message": "bump up version\n",
+        "tree": null,
+        "parents": [
+          "25422a6589feee092d7e47f12c4e7a160d086f2b",
+          "d802defdf0e6b4f85e01e48f2576b5f4ade8c77d"
+        ],
+        "url": "https://bitbucket.org/foo/bar/commits/301b16bbd02e1f2107ce3e50812afd584c5a99e0"
+      }
+    ]
```

Comment:
thanks for all this, it makes it much easier to figure out as someone who hasn't used bitbucket cloud